requirements.txt

```diff
@@ -2,58 +2,58 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements.txt setup.py requirements-python-lt-2.7.9-ssl-issue.in
+#    pip-compile --output-file=requirements.txt requirements-python-lt-2.7.9-ssl-issue.in setup.py
 #
 ansible-vault==1.1.1
-ansible==2.5.0
+ansible==2.8.0
```

Comment:
why so many requirement upgrades?

Reply:
their first commit was just `make upgrade-requirements` without the ansible version change. The 2nd commit I changed the ansible version and ran it again.